view-snp-visualizer.ftl

```diff
@@ -5,9 +5,7 @@
   <title>SNP Visualisations</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <#--work around for bug in app store, not loading resources-->
```

Comment:
should we not leave this comment and leave the module as a reminder that we need to fix the app store with respect to the loading of resources ?